superduperdb/base/config_dicts.py

```diff
@@ -10,34 +10,42 @@
 
 import fil
 
+Dict = t.Dict[str, object]
```

Comment:
Nitpick: What's the rationale for using `object` here rather than `t.Any`?

Reply:
I should possibly have changed this to be consistent, but ah, well.

In the end game, if you turn on `strict = true` (or one of the subsettings of that) then `t.Any` is disallowed anywhere, but `object` is allowed, so when I was experimenting with this file, I used `object`.